_posts/en/newsletters/2025-04-04-newsletter.md

```diff
@@ -0,0 +1,768 @@
+      merchant acceptance of bitcoins.
+
+    - *Argument from minimal benefit:* although allowing theft could be
+      used to fund the development of quantum computing, stealing coins
+      does not provide any direct benefit to honest participants in the
+      Bitcoin protocol.
+
+    - *Argument from clear deadlines:* nobody can know far in advance
+      the date at which someone with a quantum computer can begin
+      stealing bitcoins, but a specific date at which quantum-vulnerable
+      coins will be destroyed can be announced far in advance with
+      perfect precision.  That clear deadline will provide more
+      incentive for users to re-secure their bitcoins in time, ensuring fewer total coins
+      are lost.
+
+    - *Argument from miner incentives:* as noted above, quantum theft
+      would likely reduce miner income.  A persistent majority of
+      hashrate can censor spending of quantum-vulnerable bitcoins, which
+      they might do even if the rest of Bitcoiners prefer a different
+      outcome.
+
+    Lopp also provides several arguments against the destruction of
+    vulnerable bitcoins, but he concludes in favor of destruction.
+
+    Nagaev Boris [asks][boris timelock] whether UTXOs that are
+    [timelocked][topic timelocks] beyond the upgrade deadline should
+    also be destroyed.  Lopp notes existing pitfalls of long timelocks
+    and says he personally gets "a bit nervous locking funds for more
+    than a year or two."
+
+  - *Securely proving UTXO ownership by revealing a SHA256 preimage:*
+    Martin Habovštiak [posted][habovstiak gfsig] to the Bitcoin-Dev
+    mailing list an idea that could allow someone to prove they
+    controlled a UTXO even if ECDSA and [schnorr signatures][topic
+    schnorr signatures] were insecure (e.g., after fast quantum
+    computers existed).  If the UTXO contained a SHA256 commitment (or
+    other quantum-resistant commitment) to a preimage that had never
+    previously been revealed, then a multistep protocol for revealing
+    that preimage could be combined with a consensus change to prevent quantum theft.  This is
+    fundamentally the same as a [previous proposal][ruffing gfsig] by
+    Tim Ruffing (see [Newsletter #141][news141 gfsig]), which he learned
+    is generally known as the [Guy Fawkes signature scheme][].  It's
+    also a variant of a [scheme][back crsig] Adam Back invented in 2013
+    to improve resistance against miner censorship.  In short, the
+    protocol could work like this:
+
+    1. Alice receives funds to an output that, in some way, makes a
+       SHA256 commitment.  This can be a directly hashed output, such as
+       P2PKH, P2SH, P2WPKH, or P2WSH---or it can be [P2TR][topic
+       taproot] output with a script path.
+
+    2. If Alice receives multiple payments to the same output script,
+       she must either not spend any of them until she's ready to spend
+       all of them (definitely required for P2PKH and P2WSH; probably
```

Comment:
```suggestion
       all of them (definitely required for P2PKH and P2WPKH; probably
```

It is followed by `also practically required for P2SH and P2WSH`, so is this `P2WPKH` instead of `P2WSH`?